Include Demo projects in solution on build pipeline

While running the `dotnet build "Release" with version suffix` step in the build pipeline, it all seems to build without any compiler errors. However, running it locally, I get the following error:
```
  Measures.ecqm-content-qicore-2025 failed with 1 error(s) (5.5s)
    D:\firely-cql-sdk\Demo\Cql\Build\ElmToCSharp.Targets.xml(62,3): error MSB3073: The command ""D:\firely-cql-sdk\Demo\Cql\Build\../../../Cql/PackagerCLI/bin/Release/net8.0/Hl7.Cql.Packager.exe" elm  --override-utc-date-time "1970-01-01T00:00:00Z" --canonical-root-url "https://fire.ly/fhir/" --elm "D:\firely-cql-sdk\Demo\/../LibrarySets/ecqm-content-qicore-2025/Elm" --cql "D:\firely-cql-sdk\Demo\/../LibrarySets/ecqm-content-qicore-2025/Cql" --cs "D:\firely-cql-sdk\Demo\Measures.ecqm-content-qicore-2025/CSharp" --fhir "D:\firely-cql-sdk\Demo\/../LibrarySets/ecqm-content-qicore-2025/Resources" --json-pretty "true" --profile "ecqm-content-qicore-2025"" exited with code -1.
```

Find out why

--- a/Demo/Measures.Authoring/Measures.Authoring.csproj
+++ b/Demo/Measures.Authoring/Measures.Authoring.csproj
@@ -11,6 +11,8 @@
 		<ProjectReference Include="..\..\Cql\Cql.Runtime\Cql.Runtime.csproj" />
 		<ProjectReference Include="..\CLI\CLI.csproj" />
 		<PackageReference Include="Hl7.Fhir.R4" Version="$(FirelySdkVersion)" />
+		<!-- Ensure the PackagerCLI is build first, so that the ElmToCSharp targets file succeeds -->
+		<ProjectReference Include="..\..\Cql\PackagerCLI\PackagerCLI.csproj" />
 	</ItemGroup>
 
 	<PropertyGroup>

--- a/Demo/Measures.Demo/Measures.Demo.csproj
+++ b/Demo/Measures.Demo/Measures.Demo.csproj
@@ -10,6 +10,8 @@
 		<ProjectReference Include="..\..\Cql\Cql.Runtime\Cql.Runtime.csproj" />
 		<ProjectReference Include="..\CLI\CLI.csproj" />
 		<PackageReference Include="Hl7.Fhir.R4" Version="$(FirelySdkVersion)" />
+		<!-- Ensure the PackagerCLI is build first, so that the ElmToCSharp targets file succeeds -->
+		<ProjectReference Include="..\..\Cql\PackagerCLI\PackagerCLI.csproj" />
 	</ItemGroup>
 
 	<PropertyGroup>

--- a/Demo/Measures.ecqm-content-cms-2025/Measures.ecqm-content-cms-2025.csproj
+++ b/Demo/Measures.ecqm-content-cms-2025/Measures.ecqm-content-cms-2025.csproj
@@ -11,6 +11,8 @@
 		<ProjectReference Include="..\..\Cql\Cql.Runtime\Cql.Runtime.csproj" />
 		<ProjectReference Include="..\CLI\CLI.csproj" />
 		<PackageReference Include="Hl7.Fhir.R4" Version="$(FirelySdkVersion)" />
+		<!-- Ensure the PackagerCLI is build first, so that the ElmToCSharp targets file succeeds -->
+		<ProjectReference Include="..\..\Cql\PackagerCLI\PackagerCLI.csproj" />
 	</ItemGroup>
 
 	<PropertyGroup>

--- a/Demo/Measures.ecqm-content-qicore-2024/Measures.ecqm-content-qicore-2024.csproj
+++ b/Demo/Measures.ecqm-content-qicore-2024/Measures.ecqm-content-qicore-2024.csproj
@@ -11,6 +11,8 @@
 		<ProjectReference Include="..\..\Cql\Cql.Runtime\Cql.Runtime.csproj" />
 		<ProjectReference Include="..\CLI\CLI.csproj" />
 		<PackageReference Include="Hl7.Fhir.R4" Version="$(FirelySdkVersion)" />
+		<!-- Ensure the PackagerCLI is build first, so that the ElmToCSharp targets file succeeds -->
+		<ProjectReference Include="..\..\Cql\PackagerCLI\PackagerCLI.csproj" />
 	</ItemGroup>
 
 	<PropertyGroup>

--- a/Demo/Measures.ecqm-content-qicore-2025/Measures.ecqm-content-qicore-2025.csproj
+++ b/Demo/Measures.ecqm-content-qicore-2025/Measures.ecqm-content-qicore-2025.csproj
@@ -11,6 +11,8 @@
 		<ProjectReference Include="..\..\Cql\Cql.Runtime\Cql.Runtime.csproj" />
 		<ProjectReference Include="..\CLI\CLI.csproj" />
 		<PackageReference Include="Hl7.Fhir.R4" Version="$(FirelySdkVersion)" />
+		<!-- Ensure the PackagerCLI is build first, so that the ElmToCSharp targets file succeeds -->
+		<ProjectReference Include="..\..\Cql\PackagerCLI\PackagerCLI.csproj" />
 	</ItemGroup>
 
 	<PropertyGroup>

